my-app/src/test/java/com/JavaWebServer/app/GetTest.java

```diff
@@ -0,0 +1,17 @@
+package com.JavaWebServer.app;
+
+import org.junit.Test;
+import org.junit.Before;
+import static org.junit.Assert.assertEquals;
+
+public class GetTest {
+
+  @Test 
+  public void handleRequestTest() {
+    Request testrequest = new Request();
+    testrequest.setMessage("GET / HTTP/1.1");
+    Get testGet = new Get("HTTP/1.1 200 ok");
+    String response = testGet.handleRequest(testrequest);
+    assertEquals("HTTP/1.1 200 ok", response);
```

Comment:
This code may be more clear if you extract an expected response string / constant and assert against that, rather than creating the same string twice.